goal.md

```diff
@@ -24,41 +24,41 @@
 mostly because as we _age_ extreme temperatures make bones ache!
 
 
-## **Learning Center** Requirements
+## **Learning Centre** Requirements
```

Comment:
I know it _looks_ like a typo, but there's a _reason_ I spell `center` this way... 
http://ell.stackexchange.com/questions/11524/whats-the-difference-between-center-and-centre